libs/MVS/SceneTexture.cpp

```diff
@@ -1677,7 +1677,8 @@ void MeshTexture::PoissonBlending(const Image32F3& src, Image32F3& dst, const Im
 			const MatIdx idx(indices(i));
 			ASSERT(idx != -1);
 			coeffA.AddConstruct(idx, idx, 1.f);
-			coeffB[idx] = Color(dst(i));
+			Color c = dst(i);
```

Comment:
Make it unambiguous as to whether the move or copy implicit type-cast is being used (C++11 won't let you leave it ambiguous).